pkgbuild/create-installer-mac.sh

```diff
@@ -20,8 +20,15 @@ if [ -f ~/.password ]; then
   security unlock-keychain -p `cat ~/.password`
 fi
 
+set +u
+SIGN_OPTION=
+if [ ! -z "$CERTIFICATE" ]; then
+  SIGN_OPTION="--sign ${CERTIFICATE}"
+fi
+set -u
+
 cd pkgbuild
-for f in $WORKSPACE/workspace/target/OpenJDK*-j*.tar.gz;
+for f in $WORKSPACE/workspace/target/*.tar.gz;
```

Comment:
unfortunately, this will break the job, we had an issue where  it tried to create an installer for the `testimage` as well. Maybe you can add a flag for this or something?

Reply:
OK, I will add an option.